subt/main.py

```diff
@@ -299,8 +305,16 @@ def follow_wall(self, radius, right_wall=False, timeout=timedelta(hours=3), dist
                         if self.use_center:
                             desired_direction = 0
```

Comment:
This line needs to be replaced with `self.go_safely(0)`. After the changes below, go_safely() is not called anymore and ... I don't know ... the robot continues doing whatever it was doing, ignoring obstacles?

Reply:
Uhh, good one! OK, I will change and create `ver104sym4`